lib_eio_luv/eio_luv.ml

```diff
@@ -228,6 +248,23 @@ module File = struct
     await_with_cancel ~request (fun loop -> Luv.File.mkdir ~loop ~request ~mode path)
 end
 
+module Poll = struct
+
+  let poll_readable fd callback =
+    let p = Luv.Poll.init ~loop:(get_loop ()) (Obj.magic fd) in
+    match p with
+      | Ok poll ->
```

Comment:
Strange indentation here.

You can simplify this with `|> or_raise` like the rest of the code uses, which will also avoid throwing away the error message.